cmd/goProbe/config/config.go

```diff
@@ -105,11 +106,13 @@ type LogConfig struct {
 
 // APIConfig stores goProbe's API configuration
 type APIConfig struct {
-	Addr      string   `json:"addr" yaml:"addr"`
-	Metrics   bool     `json:"metrics" yaml:"metrics"`
-	Profiling bool     `json:"profiling" yaml:"profiling"`
-	Timeout   int      `json:"request_timeout" yaml:"request_timeout"`
-	Keys      []string `json:"keys" yaml:"keys"`
+	Addr                 string     `json:"addr" yaml:"addr"`
+	Metrics              bool       `json:"metrics" yaml:"metrics"`
+	Profiling            bool       `json:"profiling" yaml:"profiling"`
+	Timeout              int        `json:"request_timeout" yaml:"request_timeout"`
+	QueryMaxReqPerSecond rate.Limit `json:"query_max_req_per_sec" yaml:"query_max_req_per_sec"`
```

Comment:
Nitpick / preference: this could be its own subconfig `RateLimiting`.

Reply:
I was afraid you'd say that. Two variables is probably the magic number where adding a wrapper is just annoying. But for you I'll add it. :wink: